ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -20,7 +20,10 @@
 #include "ReactSkia/sdk/CurlNetworking.h"
 #include "ReactSkia/views/common/RSkImageUtils.h"
 #include "ReactSkia/views/common/RSkConversion.h"
+#include<mutex>
 
+#define MILLSEC_CONVERTER(time) time*1000
+mutex networkRequestMutext;
```

Comment:
usage of global mutex :  if we have multiple image components with network source url , will it not block the other threads ?
a) considering mutations are happening in sequential manner , do we need the lock 
b) already abortRequest has a lock 
c) usage of lock for bool variable , cant we use atomic bool to avoid it